src/commands/join.cpp

```diff
@@ -36,26 +37,34 @@ void Server::joinChannel(int fd, const std::string &param) {
 #ifdef DEBUG
   {
     std::ostringstream oss;
-    oss << "JOIN: client.getChannelsCount(): " << client.getChannelsCount();
+    oss << "client.getChannelsCount(): " << client.getChannelsCount();
```

Comment:
Please keep `JOIN:` keyword in log message to know what command this message comes from.